File: dev-docs/snippets/collections/delete.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Delete a collection. Returns `{ success: boolean }`.
export async function deleteX(client: JsBaoClient, collectionId: string) {
  // #region example
  const { success } = await client.collections.delete(collectionId);
  // #endregion example
  return success;
}

File: dev-docs/snippets/collections/delete.swift
import JsBaoClient

// Delete a collection. Returns `SuccessResult { success }`.
func delete(client: JsBaoClient, collectionId: String) async throws {
  // #region example
  let result = try await client.collections.delete(collectionId: collectionId)
  let success = result.success
  // #endregion example
  _ = success
}
